src/server/api/routers/files.ts

```
@@ -751,353 +552,231 @@ export const fileRouter = createTRPCRouter({
+                break;
+
+              case "blob":
+                if (!file.blobPath) {
+                  return err({
+                    status: apiErrorStatus.BadRequest,
+                    type: apiErrorTypes.BadRequestCorrupted,
+                    message: "No Path Provided",
+                  });
+                }
+    
+                const { error: del_blob_error } = await tc(del(file.blobPath));
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)